config/peerpods/podvm/ami-helper.sh

```
@@ -166,6 +169,19 @@ delete_bucket() {
 	aws s3api delete-bucket --bucket ${BUCKET_NAME} --region ${REGION}
 }
 
+delete_vmimport_role() {
+	echo "Delete vmimport role"
+	if aws iam get-role --role-name vmimport --region ${REGION} &>/dev/null; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)